source/jormungandr/jormungandr/interfaces/v1/test/elastic_search_tests.py

```diff
@@ -540,11 +540,32 @@ def bragi_make_params_with_instance_test():
     request = {"q": "aa", "count": 20}
 
     params = bragi.make_params(request=request, instances=[instance], timeout=1)
-    rsp = {'q': 'aa', 'limit': 20, 'pt_dataset[]': ['bib'], 'timeout': 1000}
-    len(list(rsp.keys())) == len(list(params.keys()))
-    for key, value in rsp.items():
-        assert key in params
-        assert value == params[key]
+    rsp = {'q': 'aa', 'limit': 20, 'pt_dataset[]': 'bib', 'timeout': 1000}
+
+    len(rsp) == len(params)
```

Comment:
?
```suggestion
    assert len(rsp) == len(params) 
```

besides, what you want is this: `assert rsp == dict(params)`, it works only when there is only one instance

AND If i understand your PR well, shouldn't `resp`  be a list ??

Reply:
oups for the `assert` :man_facepalming: 

resp is there only to test the returned values.

If I use a list for rsp I will also have sorting problems for comparing to `params` thus to dict + lookup compare.

But if you want I can also change it to a list + sort both list. what do you think ?